releasenotes/notes/generate-experiments-2ac773442132c78d.yaml

```diff
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    Added a module, :mod:`circuit_knitting.cutting.cutting_experiments`, which is intended to hold
+    functions used for generating the quantum experiments needed for circuit cutting. This module
+    will initially hold one function, :func:`.generate_cutting_experiments`,
+    which can be used to generate quantum experiments, given an input circuit containing
+    :class:`.BaseQPDGate` instances, some observables, and a number
```

Comment:
I _think_ the circuit must contain `TwoQubitQPDGate`s, specifically.

Reply:
What about wire cutting?

Reply:
Also this includes the case where the circuit is partitioned in a dictionary format, so those have SingleQubitQPDGates

Reply:
> Also this includes the case where the circuit is partitioned in a dictionary format, so those have SingleQubitQPDGates

Yes, indeed, makes sense. :)